src/liballoc/rc.rs

```diff
@@ -130,7 +130,7 @@
 //!     gadget_owner.gadgets.borrow_mut().push(gadget2.clone().downgrade());
 //!
 //!     // Iterate over our Gadgets, printing their details out
-//!     for gadget_opt in gadget_owner.gadgets.borrow().iter() {
+//!     for gadget_opt in &*gadget_owner.gadgets.borrow() {
```

Comment:
I'm not strictly sure that doing a reborrow is clearer than calling `.iter()` in cases like these... hm